packages/composables/src/getters/productGetters.ts

```diff
@@ -104,8 +104,12 @@ const getByFilters = (product: Product, filters?: ProductFilter): AgnosticProduc
     name: variant?.name,
     sku: variant?.sku,
     slug: masterVariant?.slug,
-    collections: collections?.map(collection => ({id: collection.id, name: collection.name, breadcrumbs: collection.breadcrumbs})),
-    images: [featuredAsset?.preview],
+    collections: collections?.map(collection => ({
+      id: collection.id,
+      name: collection.name,
+      breadcrumbs: collection.breadcrumbs
+    })),
+    images: [...assets.map(value => value.preview)],
```

Comment:
Maybe we should sort the images array to make sure that the `featuredAsset` is always listed in first position?